### PR TITLE
cleanup socket/plex in prep for neuron/services

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/eventbus.py [hunk 1]
```diff
@@ -70,8 +70,6 @@ def __init__(self):
 
         self._syn_links = []
 
-        self._syn_queues = {}
-
```
Comment (invisig0th) [inline synapse/eventbus.py]: this was never used anywhere....  *shrug*

File: synapse/lib/socket.py [hunk 2]
```diff
@@ -328,6 +328,7 @@ def __init__(self):
 
         #self._plex_sel = selectors.DefaultSelector()
 
+        self._plex_thr = None
```
Comment (vEpiphyte) [inline synapse/lib/socket.py]: why set this to none?

File: synapse/lib/socket.py [hunk 3]
```diff
@@ -396,6 +379,14 @@ def _popPlexSock(self, iden):
         except ValueError as e:
             pass
 
+        self.wake()
+
+    def wake(self):
+        '''
```
Comment (vEpiphyte) [inline synapse/lib/socket.py]: docstring

File: synapse/lib/socket.py (hunk 2 above)
Comment (vEpiphyte) [reply synapse/lib/socket.py]: per discussion - this is done to allow the wake functions to not fail during _init_ time.

**Pull request merged**